modules/removingBook.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Same here, Please remove disable lines for `eslint` and Fix all linters issues.